packages/conf-gmp-powm-sec/conf-gmp-powm-sec.3/opam

```diff
@@ -4,7 +4,8 @@ homepage: "http://gmplib.org/"
 bug-reports: "https://github.com/ocaml/opam-repository/issues"
 license: "GPL-1.0-or-later"
 build: [
-  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include test.c"] {os != "macos"}
+  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include test.c"] {os != "macos" & os != "win32" }
+  ["sh" "-exc" "$(ocamlc -config-var c_compiler) -c $CFLAGS test.c"] {os = "win32"}
   [
     "sh"
     "-exc"
```

Comment:
GitHub won't me suggest the change on the correct line (how many years until they'll add this, I wonder....): pedantically, the package ought to refer to the `host-arch-x86_64` etc. as it now uses them:
```diff
-depends: ["conf-gmp"]
+depends: [
+  "conf-gmp"
+  ("host-arch-x86_32" {os = "win32" & os-distribution != "cygwinports"} |
+   "host-arch-x86_64" {os = "win32" & os-distribution != "cygwinports"})
+]
```
It's pedantic because it has them as a transitive dependency.

Reply:
Applied